src/main/java/lotto/domain/LottoPurchaseBudget.java

```diff
@@ -19,17 +19,32 @@ public double calculateProfitRate(long profit) {
         return (double) profit / budget;
     }
 
+    public void purchaseLotto(int lottoCount) {
+        validatePurchasable(lottoCount);
+        budget -= (lottoCount * TICKET_PRICE);
+    }
+
+    public void validatePurchasable(int lottoCount) {
+        if (lottoCount < 0) {
+            throw new IllegalArgumentException("구입 개수는 양의 정수여야 합니다.");
+        }
+
+        if (lottoCount > getTicketQuantity()) {
```

Comment:
`purchaseLotto`에서는 유효성을 검증한 다음  `budget -= (lottoCount * TICKET_PRICE);` 로직을 수행하고있어요. 
결과적으로는 같은 의미긴 하지만 유효성 검증도 수행하는 동작과 거의 동일한 값에 대해 검증하는 것이 조금 덜 헷갈리지 않을까요? 🙂

```suggestion
        if (lottoCount * TICKET_PRICE > budget) {
```